### ISSUE TITLE
DLSS 2.x/3.x crashes games that support DLSS 1.x

### ISSUE THREAD

**Issue by AkechiiYEETS** [beeradmoore/dlss-swapper#118]
Games with DLSS 1.x should not be able to pick DLSS 2.x/3.x, games with DLSS 2.x/3.x should not be able to pick DLSS 1.x. Doing so will likely crash as reported below.

Original comment: 
> Swapping the DLSS Version from the default 1.1.13.0 to seemingly any other DLSS version crashes the game on startup

**Comment by beeradmoore** [comment]
Are you trying to swap to a 2.X version of DLSS, or is it crashing with other 1.X versions as well?

**Comment by AkechiiYEETS** [comment]
Yep, from 1x to 2x/3x, trying the latest 1x version seems to work, thought its not that big of a jump in dlss version unfortunately

**Comment by beeradmoore** [comment]
Yeah this is expected. 1 <--> 2x/3x is incompatible, 2x is compatible with 3x (keeping in mind 3 is not DLSS 3 -_-).
There was mention somewhere of just not showing 2x/3x for a game that currently has 1x (and vice versa) but I can't see that around anywhere.

Is it ok if I use this issue for tracking that task?

**Comment by AkechiiYEETS** [comment]
Thats fine

**Comment by beeradmoore** [comment]
Update on this, DLSS v1.1 will prevent users selecting DLSS 2/3 for DLSS 1 games and vice versa 

![Image](https://github.com/user-attachments/assets/2bbe0393-2360-416d-8ec9-992f5c4b43fc)

![Image](https://github.com/user-attachments/assets/e9890a6e-9f9d-4fe5-8d9c-37e56fbf670f)

**Comment by beeradmoore** [comment]
This has now been added in [DLSS Swapper v1.1](https://github.com/beeradmoore/dlss-swapper/releases/tag/v1.1)

I am closing this issue now, feel free to continue discussing in this thread. If you are still experiecing problems please open a new issue
